tools/testbuild.sh

```diff
@@ -238,6 +246,11 @@ function configure {
 function build {
   echo "  Building NuttX..."
   makefunc
+  if [ ${SAVEARTIFACTS} -eq 1 ]; then
+    artifactconfigdir=$ARTIFACTDIR/$(echo $config | sed "s/:/\//")/
+    mkdir -p $artifactconfigdir
+    cp $(${MAKE} execname) $artifactconfigdir
```

Comment:
should we copy more file here? e.g. nuttx.bin, System.map.. which is useful to debug the problem.

Reply:
Yeah but I don't know how to get those. If we can figure out how to echo, any recommendations? I figured start with the executable and refine going forward.

Reply:
For most, we have hard-coded names that has never changed.

Reply:
I'll generate a manifest file in the Makefile.(unix/win) that can be used for this.  Might need to extend the boards a bit so we know about things like uboot images.